File: internal/sast/fixtures/java/header_positive.java
// internal/sast/fixtures/java/header_positive.java
import javax.servlet.http.*;
import java.io.IOException;

public class HeaderPositive extends HttpServlet {
    public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
        resp.addHeader("X-Tracking", req.getParameter("id"));
        resp.getWriter().write("ok");
    }
}

File: internal/sast/fixtures/python/header_positive.py
# internal/sast/fixtures/python/header_positive.py
from flask import Flask, request, Response

app = Flask(__name__)

@app.route("/track")
def track():
    resp = Response("ok")
    resp.headers.set("X-Tracking", request.args.get("id", ""))
    return resp
